modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -72,6 +65,23 @@ def build_and_save_form(form)
       va_form
     end
 
+    def init_attributes(form)
+      mapped = {
+        title: form['fieldVaFormName'],
+        pages: form['fieldVaFormNumPages'],
+        language: form.dig('langcode', 'value'),
+        form_type: form['fieldVaFormType'],
+        form_usage: form.dig('fieldVaFormUsage', 'processed'),
+        form_tool_intro: form['fieldVaFormToolIntro'],
+        form_tool_url: form.dig('fieldVaFormToolUrl', 'uri'),
+        deleted_at: form.dig('fieldVaFormDeletedDate', 'value'),
+        related_forms: form['fieldVaFormRelatedForms'].map { |f| f.dig('entity', 'fieldVaFormNumber') },
+        benefit_categories: map_benefit_categories(form['fieldBenefitCategories'])
+      }
+      mapped[:form_details_url] = "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}" if form['entityPublished']
```

Comment:
Is there ever a situation where the form detail page might become unpublished?

Reply:
No, at least not for the foreseeable future. Once published, we would expect it to remain published. However, they're going to be published in batches before then so many forms (all of them are unpublished right now) will have `entityPublished == false`

Reply:
Not sure if this figures in, but it is possible that a form could be deleted (which would result in the form detail being unpublished and replaced with another form (it would get a new row id that would end up with a different form detail page that would be published later.  There is nothing technical in the CMS that keeps a form detail page from being unpublished.